app/services/food_service.rb

```diff
@@ -0,0 +1,13 @@
+require 'faraday'
+class FoodService
+  def self.connection
+    Faraday.new(url: "https://api.nal.usda.gov/fdc/v1/foods/") do |f|
+      f.params['api_key'] = 'Ze3akdRIKt8TZiohIkLEfF5va0Q5WHAshPSFaHkl'
```

Comment:
We don't want to commit our API keys. I recommend using a tool like `figaro` to help manage your keys. Then, you could call it as an environment variable here instead, like this:
` f.params['api_key'] = ENV['food_api_key']`